### ISSUE TITLE
Outdated PHPMD version

### ISSUE THREAD

**Issue by IvanChepurnyi** [codeclimate/codeclimate-phpmd#48]
PHPMD version that currently locked in composer.lock results in different results of PHP Mess Detector. 2.3 has a weird bug in the coupling between objects rule that was fixed in 2.6, but codeclimate still uses outdated version.

**Comment by ravage84** [comment]
Yes, please update to PHPMD 2.7.0, which came out at the end of July.
